charts/argo-cd/templates/argocd-configs/cluster-secrets.yaml

```
@@ -6,6 +6,9 @@ metadata:
   name: {{ include "argo-cd.name" $ }}-cluster-{{ .name }}
   labels:
     {{- include "argo-cd.labels" (dict "context" $) | nindent 4 }}
+    {{- if .labels }}
+      {{- toYaml .labels | nindent 4 }}
+    {{- end }}
```

```suggestion
    {{- with .labels }}
      {{- toYaml . | nindent 4 }}
    {{- end }}
```

I would in most cases use ``with` because you don't have to write the keypath twice.

---

Thanks, I made the update and tested locally.